server/handlers/amp-handler.js

```diff
@@ -0,0 +1,35 @@
+const { ampifyStory } = require("@quintype/amp");
+const { Story, AmpConfig } = require("../api-client");
+
+exports.handleAmpRequest = async function handleAmpRequest(
+  req,
+  res,
+  next,
+  { client, config, ampOpts }
+) {
+  try {
+    // eslint-disable-next-line no-return-await
+    const ampConfig = await config.memoizeAsync(
+      "amp-config",
+      async () => await AmpConfig.getAmpConfig(client)
+    );
+    const story = await Story.getStoryBySlug(client, req.params.slug);
+    const relatedStories = await Story.getRelatedStories(client);
+    ampConfig["related-stories"] = relatedStories;
```

Comment:
This is a race condition, please don't update the object since it's stored in cache

Reply:
Instead use {"related-stories": relatedStories, ...ampConfig.asJSON()}

Reply:
Okay, made the change